azplugins/bond.py

```diff
@@ -161,12 +161,6 @@ def process_coeff(self, coeff):
         lj1 = 4.0 * epsilon * math.pow(sigma, 12.0)
         lj2 = 4.0 * epsilon * math.pow(sigma, 6.0)
 
-        if epsilon==0:
-            hoomd.context.msg.error("azplugins.bond.fene(): epsilon must be non-zero.\n")
-            raise ValueError('epsilon must be non-zero')
-        if sigma==0:
-            hoomd.context.msg.error("azplugins.bond.fene(): sigma must be non-zero.\n")
-            raise ValueError('sigma must be non-zero')
         if k==0:
```

Comment:
Suggest to remove this too, it is not an error to have k = 0 as far as I know, although it would be silly.